openy.install

```diff
@@ -1144,3 +1147,10 @@ function openy_update_8081() {
     ]);
   }
 }
+
+/**
+ * Enable openy_block_branch_contacts_info module.
```

Comment:
The comment doesn't describe the thing.

Reply:
Fixed in https://github.com/ymcatwincities/openy/pull/2179/commits/b0d332dbd25b89d700d63f8eff66a4d457d46594